src/misc.asm

```diff
@@ -32,6 +36,22 @@ org $828BB3
     JSL gamemode_end
 
 
+; Replace unnecessary logic checking controller input to toggle debug CPU brightness
+; with logic to collect the v-counter data
+org $828AB1
+    %a8() : LDA $4201 : ORA #$80 : STA $4201 : %ai16()
+    LDA $2137 : LDA $213D : STA !ram_vcounter_data
```

Comment:
This logic was moved.  In the new location it is overwriting a rep 30 (that's why it now has %ai16 instead of %a16), so that saves one instruction.  I also removed an AND #$00FF since that only needs to be done if we draw the CPU usage.

This saves 6 CPU cycles every frame